CHANGELOG.md

```diff
@@ -6,11 +6,17 @@
 
 - Added an optional second parameter to the `include` tag for passing a sub context to the included file.  
   [Yonas Kolb](https://github.com/yonaskolb)
-  [#394](https://github.com/stencilproject/Stencil/pull/214)
+  [#214](https://github.com/stencilproject/Stencil/pull/214)
+- Variables now support indirect evaluation. For example, if you have a variable `key = "name"`, and an
```

Comment:
This would be a breaking change right? If so then it should go into [`### Breaking` section](https://github.com/kylef/changelog/blob/master/Specification.md#section) too.

For example, the following that works in Stencil 0.11 would no longer work?:

```swift
let schema = [
  "$schema": "http://json-schema.org/draft-06/schema#",
  "type": "object",
  "description" "A geographical coordinate",
  "properties": [
    "latitude": ["type": "number"],
    "longitude": ["type": "number"],
  ]
]
```

```html+django
<h2>{{ schema.description|default:"Schema" }}</h2>
<a href="{{ schema.$schema }}">View Schema Specification</a>

<code>
  {{ schema|renderJSON }}
</code>
```

---

Perhaps a better alternative would be to make this a filter itself instead of overloading special sub-syntax into this, then the same can be achieved by the following syntax which perhaps it a little clearer for the reader of the template:

```html+django
{{ item|valueForKey:"key" }}
```

Reply:
We had a pr for that actually #191, but then I added support for single parameter for `map` filter in #189 , for which we can also define `valueForKey` synonym.

Reply:
Yeah, I'm rewriting it to use `object[property]` syntax like yinya.